.github/workflows/release.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   release:
     uses: itzg/github-workflows/.github/workflows/go-with-releaser-image.yml@main
     with:
-      go-version: "1.20.10"
+      go-version: "1.21.10"
```

Comment:
It was fixed in 1.20.11. Why bump the entire minor version?

Reply:
The first two in the list will be fixed by Golang v1.20.11.

However next 7 will be a longer term patch using +v1.21.9 so I used the latest stable in that minor version set 
![image](https://github.com/itzg/rcon-cli/assets/70344584/ceff29f4-79f4-4fc9-8bb4-5b18f6b86cd6)

https://scout.docker.com/vulnerabilities/id/CVE-2024-24785/org/camcast3?s=golang&n=stdlib&t=golang&vr=%3C1.21.8&utm_source=desktop&utm_medium=ExternalLink

Reply:
I can move to 1.20.14 if you are more comfortable with that

Reply:
That all sounds good. I was focused on the major CVE item.